README.yaml

```diff
@@ -47,7 +47,7 @@ usage: |-
   ```
   FROM cloudposse/helmfiles:latest as helmfiles
 
-  COPY --from helmfiles /helmfile.d/ /conf/helmfiles.d/
+  COPY --from helmfiles /releases/ /conf/releases/
```

Comment:
What do think about using the `/conf/helmfiles/releases` pattern?

Reply:
so there would be a `/conf/helmfiles/scripts` and `/conf/helmfiles/helmfile.yaml`

Reply:
Agree